app/src/main/java/org/systers/mentorship/viewmodels/SignUpViewModel.kt

```diff
@@ -0,0 +1,73 @@
+package org.systers.mentorship.viewmodels
+
+import android.arch.lifecycle.MutableLiveData
+import android.arch.lifecycle.ViewModel
+import android.util.Log
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.annotations.NonNull
+import io.reactivex.observers.DisposableObserver
+import io.reactivex.schedulers.Schedulers
+import org.systers.mentorship.MentorshipApplication
+import org.systers.mentorship.R
+import org.systers.mentorship.remote.datamanager.AuthDataManager
+import org.systers.mentorship.remote.responses.CustomResponse
+import org.systers.mentorship.remote.responses.RegisterRequest
+import org.systers.mentorship.utils.CommonUtils
+import retrofit2.HttpException
+import java.io.IOException
+import java.util.concurrent.TimeoutException
+
+/**
```

Comment:
I didn't create this 😉

Reply:
copy and paste traces ;) will remove that!